client/src/components/cascadeSite/Slides/Slides.scss

```diff
@@ -92,10 +93,33 @@
   position: relative;
   top: 0;
   left: 0;
-  width: 100%;
   z-index: 2;
-  //
-  color: white;
+  margin: 0 1rem;
+  div {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    width: fit-content;
+    background-color: rgba(255, 255, 255, 0.7);
+  }
+  h1 {
+    font-size: 3em;
+    color: black;
+    max-width: 70rem;
+    padding: 1rem;
+    width: fit-content;
+    text-transform: uppercase;
+    background-color: rgba(255, 255, 255, 0.7);
+  }
+  p {
+    font-size: 1.4em;
```

Comment:
Will probably need to adjust these for mobile.

Reply:
I can add some font sizes for mobile and desktop.